### PR TITLE
Added Banner component and tests

### PR DESCRIPTION
**Pull request by krischarbonneau**
# Description

[Canada.ca style banner image](https://lj-decd.atlassian.net/browse/LJ-88?atlOrigin=eyJpIjoiZDNhY2ZkODY5NjY4NGJkOGJkNDRlYjRmMTgwNDlhODgiLCJwIjoiaiJ9)

Added a Canada.ca style banner component

## Test Instructions

1. Pull in branch
2. Type ```npm run dev```
3.  Navigate to [http://localhost:3000/having-a-child](http://localhost:3000/having-a-child)

## Meets Definition of Done

- [x] Meets design spec
- [x] unit tests are included

### CODE REVIEW

File: components/layout.js [hunk 1]
```diff
@@ -21,6 +22,10 @@ export default function Layout({ children }) {
 
       <Header />
 
+      {bannerText && bannerTitle ? (
+        <Banner siteTitle={bannerTitle} headline={bannerText} />
+      ) : null}
```
Comment (pcraig3) [inline components/layout.js:27]: This is kind of weird, I assumed we would pass in the component itself, not just pass attributes to layout. Could we not just pass it in on the new page?

File: pages/lifejourney/havingAChild.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import Layout from "../../components/layout";
+
+export default function lifejourney() {
+  return (
+    <Layout
+      bannerTitle="Having a child"
+      bannerText="Lorem ipsum dolor sit amet, consectetur"
+    ></Layout>
+  );
+}
```
Comment (pcraig3) [inline pages/lifejourney/havingAChild.js:10]: We _should_ include a new page here, but probably don't need 2 levels of url nesting. Can we just put this page at `pages/having-a-child.js`?

**Review by pcraig3 — COMMENTED** [review]
Looks really good! Just a couple comments.

File: components/layout.js (hunk 1 above)
Comment (krischarbonneau) [reply components/layout.js:27]: This was how Alpha implemented it, tried doing it the way you mentioned prior to doing it this way and it was doing some funky stuff with the positioning/size of the banner (either being too small or off to the right side of the screen).

File: pages/lifejourney/havingAChild.js (hunk 2 above)
Comment (krischarbonneau) [reply pages/lifejourney/havingAChild.js:10]: No issue changing this, followed the folder structure that Pond was using for the breadcrumbs component but can absolutely change it to the above.

**Review by P0NDER0SA — APPROVED** [review]
with Pauls minor comments, i feel like the teacher with an easy A - but this looks pretty clean.

**Review by pcraig3 — APPROVED** [review]
Approved! Good work 👍

**Pull request merged**